app/routes/matador/$queueName/index.tsx

```diff
@@ -1,14 +1,16 @@
 import { Divider, Grid, Title } from "@mantine/core";
 import type { LoaderFunction } from "@remix-run/node";
-import { useLoaderData } from "@remix-run/react";
+import { useCatch, useLoaderData } from "@remix-run/react";
 import React, { useState } from "react";
 import { JobsTable } from "~/lib/matador/components/jobs-table";
 import { StatCard } from "~/lib/matador/components/stat-card";
 import type { BullJob } from "~/lib/matador/index.server";
 import { getQueueJobs, getQueues } from "~/lib/matador/index.server";
-import type { JobStatus} from "~/lib/matador/types/JobStatus";
+import type { JobStatus } from "~/lib/matador/types/JobStatus";
 import { JobStatuses } from "~/lib/matador/types/JobStatus";
 
+import { ErrorFallback } from "~/lib/matador/components/error";
```

Comment:
Is there a blank line?

Reply:
Fixed with f089a705ecee1a85e45b3512238c5af13eef525a